admin_guide/ui_extensions.adoc

```diff
@@ -0,0 +1,154 @@
+= Web Console Extensions
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Loading Custom Scripts and Stylesheets
+Administrators can customize the
+link:../architecture/infrastructure_components/web_console.html[web console]
+using extensions, which let you run scripts and load custom stylesheets when
+the web console loads. You can change the look and feel of nearly any aspect of
+the user interface in this way.
+
+To add scripts and stylesheets, edit the
+link:master_node_configuration.html[master configuration]. The files must exist
+on the Asset Server and are added with the following options:
+
+----
+assetConfig:
+  extensionScripts:
+    - extensions/script1.js
+    - extensions/script2.js
+  extensionStylesheets:
+    - extensions/stylesheet1.css
+    - extensions/stylesheet2.css
+----
+
+Relative paths are resolved relative to the master configuration file. To pick
+up configuration changes, restart the server.
+
+Generally, the web console only reads custom scripts and stylesheets once on
```

Comment:
Custom scripts and stylesheets are read once at server start time. To make developing extensions easier, you can reload scripts and stylesheets on every request by enabling development mode with the following setting:

```
----
assetConfig:
  ...
  extensionDevelopment: true
----
```